custom/icds_reports/README.md

```diff
@@ -207,3 +207,17 @@ The difference is cancel is SIGTERM and terminate is SIGKILL described in more d
 Stopping all ucr_indicator_queue to reduce load: `cchq icds fab supervisorctl:"stop commcare-hq-icds-celery_ucr_indicator_queue_0"`
 
 Purging all aggregation tasks from the queue: `./manage.py celery amqp queue.purge icds_aggregation_queue`
+
+Restarting the aggregation *before doing this you should be certain that the aggregation is not running and will not start on its own*:
```

Comment:
This is already merged so fine for now, but I would say that it is important to check pg to figure out if the aggregation is running. Celery reporting on active tasks, even with our management commands can be flaky